types/mongorito/index.d.ts

```diff
@@ -0,0 +1,297 @@
+// Type definitions for mongorito v3.0.4
+// Project: https://github.com/vadimdemedes/mongorito
+// Definitions by: Pinguet62 <https://github.com/pinguet62>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import {Collection, CommonOptions, Db, IndexOptions, Long, MongoClientOptions, ReadPreference} from 'mongodb';
+
+declare namespace mongorito {
+}
```

Comment:
It seems the empty namespace can be removed

Reply:
removed